pymc3/distributions/continuous.py

```diff
@@ -2642,8 +2655,13 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
+        # To avoid issue with #4340
+        safe_alpha = tt.switch(tt.lt(alpha, 0), 0, alpha)
+        safe_beta = tt.switch(tt.lt(beta, 0), 0, beta)
```

Comment:
Again, I would advise against silently modifying user input.
Better raise `ValueError` is the user is being unreasonable with a message explaining why.